Auth.md

```diff
@@ -0,0 +1,19 @@
+# chatserver_docs
+All docs and design specs for our client-server based chat application to rule the world!
+
+### AUTHENTIFICATION METHOD
```

Comment:
header is a bit inconsistent. You use # at the top, but here ###. This should probably be only ##.